Makefile

```diff
@@ -39,7 +39,8 @@ checks:
 run-build: deps checks clean
 	cd Godeps/_workspace/src/github.com/contiv/ && chmod -R 771 contivmodel/ && cd contivmodel/ && ./generate.sh && \
 	cd /opt/gopath/src/github.com/contiv/netplugin && \
-	godep go install -v $(TO_BUILD) 
+	godep go install -v $(TO_BUILD) && \
+	sudo cp contrib/completion/bash/netctl /etc/bash_completion.d/netctl
```

Comment:
You might want to make the bash completion script part of the netplugin release tar file. Please see "make tar" target. That way, you can use the file in ansible scripts to setup bash completion.

You can do this as a separate PR if you wish.